extract.py

```diff
@@ -166,7 +269,23 @@ def write_data(output_rows, args):
 
 def main():
     args = parse_arguments()
-    output_rows = extract_database(args)
+    print()
+    print("csv-conf:", args.csv_conf)
```

Comment:
```suggestion
```
This looks like it just prints out the filename, which is ok for debugging but not super useful long term